raiden/network/transport/matrix.py

```diff
@@ -711,15 +755,15 @@ def _send_raw(self, receiver_address, data):
         room = self._get_room_for_address(receiver_address)
         if not room:
             return
-        self.log.debug('SEND', room=room, data=data)
+        self.log.debug('SEND', receiver=pex(receiver_address), room=room, data=data)
```

Comment:
btw, if you can, adding the current node address is quite useful for debugging and we have it done one a good number of logging calls. if you don't mind,next time please add the equivalent of `node=pex(node_address)`

Reply:
This was avoided in `matrix` explicitly because `self.log` is bound to include `current_user=<user_id>`, which is always in the format `@<hex_addr>:<server_hostname>`, which _replaces_ `pex(node_address)` with more information, so there wasn't the need to spam the log entry with redundant information

Reply:
I don't agree, the **only** reason to use structlog is to have parseable logs, which can then be added to a database and queried. If the logs don't have consistent fields to query the data is less useful.

Just be consistent with the rest of the code base. 13 letters in a log line that has the whole message body is hardly "spamming" anything.

Reply:
The fields **are** consistent IMO, all of them include the node address or part of it. Matrix isn't supposed to comply with **conventions** from the Service, it is (or was made to be) a standalone class. If you want to require a given field to be on every entry, you should consider making a PR to `log.bind` the node address in `raiden_service.py`, and import/require/inject/whatever that on every child logger.
Anyway, I don't have hard feelings on this one, agree with you that 13 characters is negligible, so I'll bind and include it here, but above suggestion holds.